source/docs/advanced/performance/assets-cdn-domain.md

```diff
@@ -0,0 +1,35 @@
+---
+id: assets-cdn-domain
+title: Serving assets from a CDN/custom domain
+---
+
+<blockquote class="feature--new">
+  _This feature has been added in version `2.7.0`_
+</blockquote>
+
+## Configuration
+
+Serving assets from a custom domain can be done with a simple configuration:
```

Comment:
Nit: until now, we've tried to avoid adjectives such as "just", "easy" and "simple" in the documentation for making it more readable and inclusive ([some](http://jackkelly.name/blog/archives/2019/09/20/avoid_just_and_simply_in_documentation/index.html) random [posts](https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher) about it)
```suggestion
Serving assets from a custom domain can be done with a configuration change:
```